sphinx/source/docs/user_guide/embed.rst

```diff
@@ -370,33 +371,32 @@ Then inserting the script and div elements into this boilerplate:
         </body>
     </html>
 
-Note that above we have not included the ``"-widgets"`` JS and CSS files, since the
-document does not use Bokeh widgets. If required, the CDN resources are available as HTTPS
-URLs as well.
+Note that this doesn't include JavaScript and CSS files for ``"-widgets"``
+because the document doesn't use any Bokeh widgets.
+
+CDN resources are also available as HTTPS URLs if required.
```

Comment:
let's make sure we demo only HTTPS URLS throughout this page and then delete this sentence